circuits/aes-gcm/aes-gcm.circom

```diff
@@ -71,21 +71,16 @@ template AESGCM(l) {
 
 
     // Step 4: Let u and v (v is always zero with out key size and aad length)
-    var u = 128 * (l \ 128) - l;
     var blockCount = l\16;
     if(l%16 > 0){
         blockCount = blockCount + 1;
     }
-    var ghashblocks = 1 + blockCount + 1; // blocksize is 16 bytes
```

Comment:
but the 1's are homoiconic

Reply:
haha, It took me a while to dissect why there was three of them